src/utils.js

```diff
@@ -0,0 +1,12 @@
+function findHighest(a,b,c) {
+    if(a > b && a > c)
+      return a;
+    if(b > a && b > c)
+      return b;
+    if(c > a && c > b)
+      return c;    
```

Comment:
What happens if a,b and c are all the same values. what will be returned?

Reply:
I will look into that